src/HelperPluginManager.php

```diff
@@ -256,7 +256,7 @@ class HelperPluginManager extends AbstractPluginManager
         Helper\EscapeHtml::class          => InvokableFactory::class,
         Helper\EscapeHtmlAttr::class      => InvokableFactory::class,
         Helper\EscapeJs::class            => InvokableFactory::class,
-        Helper\EscapeCss::class           => InvokableFactory::class,
+        Helper\EscapeCss::class           => Helper\Service\EscapeCssFactory::class,
```

Comment:
All of the helpers are still new-able without a factory. Should factory addition be delayed for the inclusion of a Config Provider in #121 or for v3?